### PR TITLE
v1 of reaper script

### PR DESCRIPTION
**Pull request by mikahpinegar**
- create reaper shell script to remove uploaded folders and add logging

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -158,4 +158,12 @@ cython_debug/
...
\ No newline at end of file
+*.ARW
+
+# log files
+mini_computer_api/files_uploaded_to_globus.txt
+mini_computer_api/deleted_files.log
+
+# image folders
+/mini_computer_api/images/*
+/mini_computer_api/test_images/*
```
Comment (priyajakhar) [inline .gitignore]: Resolve the conflicts and I don't think we need this directory (/test_images).

File: mini_computer_api/main.py [hunk 2]
```diff
@@ -14,7 +14,7 @@
 logfile = str(log_file_path)
 logging.basicConfig(filename=logfile, filemode="a", format="[ %(asctime)s ] %(message)s ", datefmt="%m-%d-%y %H:%M:%S", level=logging.INFO)
 
-STATE = 'NC'
+STATE = 'MD'
```
Comment (priyajakhar) [inline mini_computer_api/main.py]: Should we be keeping this change?

Comment (mikahpinegar) [reply mini_computer_api/main.py]: this is what made me create #61

File: .gitignore (hunk 1 above)
Comment (mikahpinegar) [reply .gitignore]: good catch i will delete that directory and remove from gitignore

**Pull request merged**